docs/siem/detections/api/signals-api-overview.asciidoc

```diff
@@ -2,17 +2,17 @@
 [role="xpack"]
 === Signals endpoint
 
-The signals endpoint is for retrieving, aggregating, and updating signals. 
-For detailed information on how to retrieve and aggregate results from the 
-indices, see:
+The signals endpoint is for retrieving, aggregating, and updating detection
+alerts. For detailed information on how to retrieve and aggregate results from
+the indices, see:
 
 * {ref}/getting-started-search.html[Start searching]
```

Comment:
"Start searching" seems general? Is this for site search? Searching through signals?